v5/pivotal/util.go

```diff
@@ -0,0 +1,90 @@
+/*
+
+   Copyright (C) 2015 Scott Devoid
+
+*/
+package pivotal
+
+import (
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+// cursor tracks response headers from paginated API responses.
+// And sets the appropriate URI variables in the next request.
+type cursor struct {
+	client *Client
+	req_fn func() (req *http.Request)
```

Comment:
@devoid I don't really like this naming convention, please use CamelCase, like `reqFunc` or something...